<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zec-qt-wallet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../application.qrc">
    <normaloff>:/icons/res/icon.ico</normaloff>:/icons/res/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Balance</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Summary</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Shielded</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="balSheilded">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Transparent</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="balTransparent">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="Line" name="line">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Total</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="balTotal">
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="unconfirmedWarning">
                 <property name="styleSheet">
                  <string notr="true">color: red;</string>
                 </property>
                 <property name="text">
                  <string>Some transactions are not yet confirmed</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Address Balances</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTableView" name="balancesTable">
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="cornerButtonEnabled">
                <bool>false</bool>
               </property>
               <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Send</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Pay From</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QComboBox" name="inputsCombo"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Address Balance</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="sendAddressBalance">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sendAddressBalanceUSD">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Send To</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QScrollArea" name="sendToScrollArea">
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="sendToWidgets">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>841</width>
                <height>321</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="sendToLayout">
               <item>
                <widget class="QGroupBox" name="verticalGroupBox">
                 <property name="title">
                  <string>Recipient</string>
                 </property>
                 <layout class="QVBoxLayout" name="sendAddressLayout">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_12">
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>Address</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="Address1">
                      <property name="placeholderText">
                       <string>Address</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_13">
                    <item>
                     <widget class="QLabel" name="label_6">
                      <property name="text">
                       <string>Amount</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="Amount1">
                      <property name="baseSize">
                       <size>
                        <width>200</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                      </property>
                      <property name="placeholderText">
                       <string>Amount</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="AmtUSD1">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="Max1">
                      <property name="text">
                       <string>Max Available</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_4">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QPushButton" name="MemoBtn1">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                      <property name="toolTip">
                       <string/>
                      </property>
                      <property name="text">
                       <string>Memo</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QLabel" name="MemoTxt1">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="addAddressButton">
                   <property name="baseSize">
                    <size>
                     <width>200</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Add Address</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Miner Fee:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblMinerFee">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblMinerFeeUSD">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblDevFeeTxt">
            <property name="text">
             <string>Dev Fee:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblDevFee">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblDevFeeUSD">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="sendTransactionButton">
            <property name="baseSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Send</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelSendButton">
            <property name="baseSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Receive</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Address Type</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QRadioButton" name="rdioZSAddr">
                 <property name="text">
                  <string>z-Addr(Sapling)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rdioZAddr">
                 <property name="text">
                  <string>z-Addr(Sprout)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rdioTAddr">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>t-Addr</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QComboBox" name="listRecieveAddresses">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="duplicatesEnabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnRecieveNewAddr">
                 <property name="text">
                  <string>New Address</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QPlainTextEdit" name="txtRecieve">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="qrcodeDisplay">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #fff</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Transactions</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="transactionsTable">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBalance">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Private_Key"/>
    <addaction name="actionTurnstile_Migration"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDonate"/>
    <addaction name="actionCheck_for_Updates"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuBalance"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check github.com for Updates</string>
   </property>
  </action>
  <action name="actionTurnstile_Migration">
   <property name="text">
    <string>Sapling Turnstile</string>
   </property>
  </action>
  <action name="actionImport_Private_Key">
   <property name="text">
    <string>Import Private Key</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>inputsCombo</tabstop>
  <tabstop>sendAddressBalance</tabstop>
  <tabstop>sendToScrollArea</tabstop>
  <tabstop>Address1</tabstop>
  <tabstop>Amount1</tabstop>
  <tabstop>Max1</tabstop>
  <tabstop>addAddressButton</tabstop>
  <tabstop>sendTransactionButton</tabstop>
  <tabstop>cancelSendButton</tabstop>
  <tabstop>balancesTable</tabstop>
  <tabstop>rdioZAddr</tabstop>
  <tabstop>rdioTAddr</tabstop>
  <tabstop>listRecieveAddresses</tabstop>
  <tabstop>btnRecieveNewAddr</tabstop>
  <tabstop>txtRecieve</tabstop>
  <tabstop>transactionsTable</tabstop>
 </tabstops>
 <resources>
  <include location="../application.qrc"/>
 </resources>
 <connections/>
</ui>
